indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -40,13 +40,15 @@
         var categoryRowSelector = 'tr[data-category-id]';
         var checkboxSelector = 'input[name=category_id]';
 
+
         $('.js-sort-categories').on('click', function() {
+            var sort_order = $(this).data('sort-order');
```

Comment:
I think @ThiefMaster will complain that we normally use `sortOrder` (jQuery does the conversion from camelCase internally).